scripts/helmer.sh

```diff
@@ -139,6 +139,12 @@ do_install() {
         retry "kubectl get -n $NAMESPACE deployment/policy-server-default" 5
         wait_rollout -n $NAMESPACE deployment/policy-server-default
     fi
+
+    # Enforce PSA restricted profile - https://github.com/kubewarden/helm-charts/pull/648
+    if is_version ">1.22" "${vMap[app]}"; then
+        kubectl label ns $NAMESPACE kubewarden.io/psa-restricted=restricted --overwrite
```

Comment:
Could we move this before doing the `helm_up` of charts, together with a `kubectl create ns $NAMESPACE`?

I think it's cleaner, so we don't have pods/containers in error state.